examples/data-sources/powerscale_snapshot_schedule/datasource.tf

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright (c) 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+# PowerScale SMB shares provide clients network access to file system resources on the cluster
+
+# Returns a list of all the PowerScale Snapshot schedules
+data "powerscale_snapshot_schedule" "example_snapshot_schedule_all" {
+  filter {
+
+    # Used for query parameter, supported by PowerScale Platform API
+    # dir = "ASC"
```

Comment:
Can you add a description to these and what they are filtering on?

Reply:
@doriac11 Updated with description.